src/main/java/com/internet/mixshop/util/ConnectionUtil.java

```diff
@@ -0,0 +1,31 @@
+package com.internet.mixshop.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String URL = "jdbc:mysql://localhost:3306/mix?serverTimezone=UTC";
+
+    static {
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
```

Comment:
Intellij Idea says that this is deprecated and we should write `"com.mysql.cj.jdbc.Driver"`

Reply:
done